Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian
+FROM alpine:3.5
```

Comment:
what version of libc does this image use? we've had DNS-related issues in the past w/ non-glibc impls. newer versions of golang w/ DNS resolution "baked in" may mitigate this concern